lib/suspenders/app_builder.rb

```diff
@@ -253,7 +253,7 @@ def configure_simple_form
     end
 
     def configure_action_mailer
-      action_mailer_host "development", %{"localhost:#{port}"}
+      action_mailer_host "development", %{"localhost:3000"}
```

Comment:
I cannot use `%{ENV.fetch("APPLICATION_HOST")}` because when we run `suspenders app_name` for the first time `.env` does not exists yet (it will be created only executing `bin/setup.sh`).

I don't known if `localhost:3000` is already the Rails default for development environment.